src/pluto/pluto.py

```diff
@@ -281,7 +281,14 @@ def generate_graph(v1v2, v2v3, v1v2_total, v2v3_total, group):
                 fig.update_layout(title=group["title"], template="plotly_white")
                 st.plotly_chart(fig)
                 st.info(group["description"])
+
             st.write(df)
+            st.info(
```

Comment:
Don't think this info is clearly associated with the table. The table could use it's own header that clearly breaks it off into it's own section

Reply:
👍